Code of conduct (#1)
mietright/code-of-conduct: README.md

# Code of Conduct

## Diversity
We accept others, no matter the gender, nationality, culture, religion.

## Respect

> We all have different backgrounds, beliefs and points of view.

Reviewer: ```suggestion
We all have different backgrounds, beliefs, and points of view. We respect all of them.
```

> We all have different backgrounds, beliefs and points of view.

Reviewer: This feels like the duplicate of the first one.

Author: Maybe, but we should remark both of them. Maybe we should add something else here.

Treat others as you wish to be treated.

## Trust and Cooperation

> We are a team, don't be afraid to be judged. We support each other.

Reviewer: ```suggestion
We are a team, we don't judge rather support each other.
```

## Autonomy

> You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.

Reviewer: ```suggestion
We are the owner of our tasks, and we choose how to do our tasks, when to ask for help, and when to pull the plug on a discussion that we asked for.
```

Reviewer 2: Why we? `We are the owner` doesn't make much sense. What do you want to say here?

Reviewer: I think You tense is just wrong. I know my suggestion also doesn't fix it completely. But something is wrong here. The sentence has to be more elegant. It is rude right now.

> ## Responsability

Reviewer 3: ```suggestion
## Responsibility
```

> You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.

Reviewer: ```suggestion
We are individually responsible for our parts of the products. Also, collectively as a team, we all are responsible for the health and features of whole product.
```

## Ownership

> Be the owner of your code and projects. If something fails on it, you have to take care of it.

Reviewer: Does that mean who ever created the bug is responsible to fix it?

> Be the owner of your code and projects. If something fails on it, you have to take care of it.

Reviewer: ```suggestion
Each on of us is the owner of their code and/or projects. If something fails, they have to take care of the failure, ensure completeness and full health of it.
```

> Be the owner of your code and projects.  If something fails on it, you have to take care of it.

Reviewer 3: ```suggestion
Be the owner of your code and projects. If something fails on it, you have to take care of it.
```

> ## Disagree

Reviewer 3: ## Healthy Culture of Controversy
In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.

You don't have to agree with everything we say, it is ok to disagree, give your arguments and let's set up an agreement.

> ## Commit

Reviewer 3: ## Disagree and Commit
It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discus it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.

> In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Additionally the person who works on the task has the right to stop the argument at any time, take an informed decision and continue working.

Reviewer: ```suggestion
In a discussion it is ok to disagree. But a healthy culture of controversy includes, that other points of views next to yours are not only allowed, but as valid and valuable as your own. Though the person who works on the task has the absolute right to stop the argument at any time.
```

> Let's commit on what we agree. (please, I need help to finish this one)

Author: ```suggestion
It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discus it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.
```

> It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discuss it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.

Reviewer: ```suggestion
It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discuss it and make your points during discussion, but once committed you are expected to deliver the highest quality work, specially in broader and shared tasks.
```

## Raise the flag

> If you feel that there is something wrong, please raise the flag and speak. Let's have a meeting and talk about the things that you don't like. Your thoughts are important for us.

Reviewer: ```suggestion
If we feel that something is wrong, we should not be afraid to raise the flag and speak about it. 
We have talk about the things that we don't like. Everyone is equally important for us.
```

# Code of Conduct

## Diversity
We accept others, no matter the gender, nationality, culture, religion.

## Respect
We all have different backgrounds, beliefs, and points of view. We respect all of them.
Treat others as you wish to be treated.

## Trust and Cooperation
We are a team, we don't judge rather support each other.

## Autonomy

> You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.

Reviewer 3: ```suggestion
Each member of the team is the owner of their task. They choose how to do the tasks , when to ask for help, and when to pull the plug on a discussion that they asked for.
```

> Each member of the team is the owner of their task. They choose how to do the tasks , when to ask for help, and when to pull the plug on a discussion that they asked for.

Reviewer: ```suggestion
Each member of the team is the owner of their tasks. They choose how to do their tasks, when to ask for help, and when to pull the plug on a discussion that they asked for.
```

## Responsibility
We are individually responsible for our parts of the products. Also, collectively as a team, we all are responsible for the health and features of the whole product.

## Ownership
Each one of us is the owner of their code and/or projects. If something fails, they have to take care of the failure, ensure completeness and ensure the return to full health.

## Healthy Culture of Controversy

> It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's views might be equally valid and valuable. Although the person who works on the task gets to have the absolute right to stop the argument at any time.

Reviewer 3: ```suggestion
It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's views are equally valid and valuable. Although the person who works on the task gets to have the absolute right to stop the argument at any time.
```
